From 1afecfa2adcee0459c0b06e5a5463be4281e3df0 Mon Sep 17 00:00:00 2001
From: "Li, Guangli" <guangli.li@intel.com>
Date: Fri, 23 Aug 2019 23:12:27 +0800
Subject: [PATCH 2/9] Subject: [PATCH] Fix renamed buffer constants

Signed-off-by: Li, Guangli <guangli.li@intel.com>
Signed-off-by: Sean V Kelley <sean.v.kelley@linux.intel.com>
---
 source/blender/blenkernel/intern/writeffmpeg.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/source/blender/blenkernel/intern/writeffmpeg.c b/source/blender/blenkernel/intern/writeffmpeg.c
index f3336ad..c03810d 100644
--- a/source/blender/blenkernel/intern/writeffmpeg.c
+++ b/source/blender/blenkernel/intern/writeffmpeg.c
@@ -709,7 +709,7 @@ static AVStream *alloc_video_stream(FFMpegContext *context,
 
   if ((of->oformat->flags & AVFMT_GLOBALHEADER)) {
     PRINT("Using global header\n");
-    c->flags |= CODEC_FLAG_GLOBAL_HEADER;
+    c->flags |= AV_CODEC_FLAG_GLOBAL_HEADER;
   }
 
   /* xasp & yasp got float lately... */
@@ -840,7 +840,7 @@ static AVStream *alloc_audio_stream(FFMpegContext *context,
   }
 
   if (of->oformat->flags & AVFMT_GLOBALHEADER) {
-    c->flags |= CODEC_FLAG_GLOBAL_HEADER;
+    c->flags |= AV_CODEC_FLAG_GLOBAL_HEADER;
   }
 
   set_ffmpeg_properties(rd, c, "audio", &opts);
@@ -859,14 +859,14 @@ static AVStream *alloc_audio_stream(FFMpegContext *context,
   st->codec->time_base.den = st->codec->sample_rate;
 
 #  ifndef FFMPEG_HAVE_ENCODE_AUDIO2
-  context->audio_outbuf_size = FF_MIN_BUFFER_SIZE;
+  context->audio_outbuf_size = AV_INPUT_BUFFER_MIN_SIZE;
 #  endif
 
   if (c->frame_size == 0) {
     // used to be if ((c->codec_id >= CODEC_ID_PCM_S16LE) && (c->codec_id <= CODEC_ID_PCM_DVD))
     // not sure if that is needed anymore, so let's try out if there are any
     // complaints regarding some ffmpeg versions users might have
-    context->audio_input_samples = FF_MIN_BUFFER_SIZE * 8 / c->bits_per_coded_sample / c->channels;
+    context->audio_input_samples = AV_INPUT_BUFFER_MIN_SIZE * 8 / c->bits_per_coded_sample / c->channels;
   }
   else {
     context->audio_input_samples = c->frame_size;
-- 
2.22.0

