From 573c0134a5b271d48a170c2781d69a332cb70b59 Mon Sep 17 00:00:00 2001
From: "Li, Guangli" <guangli.li@intel.com>
Date: Mon, 26 Aug 2019 17:39:40 +0800
Subject: [PATCH] [Subject]: scripts.patch

Signed-off-by: Li, Guangli <guangli.li@intel.com>
Signed-off-by: Sean V Kelley <sean.v.kelley@linux.intel.com>
---
 source/creator/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 12046c0..be88686 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -376,8 +376,8 @@ if(WITH_PYTHON)
   endif()
 
   install(
-    DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts
-    DESTINATION ${TARGETDIR_VER}
+    DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts/
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts/
     PATTERN ".git" EXCLUDE
     PATTERN ".gitignore" EXCLUDE
     PATTERN ".arcconfig" EXCLUDE
-- 
2.22.0

