From 07d6543bd262f84ed3d1373b90069ec6b35defa9 Mon Sep 17 00:00:00 2001
From: "Li, Guangli" <guangli.li@intel.com>
Date: Sat, 24 Aug 2019 00:45:10 +0800
Subject: [PATCH] openvdb3-abi

Signed-off-by: Li, Guangli <guangli.li@intel.com>
From: Sean V Kelley <sean.v.kelley@linux.intel.com>
---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f294b57..1b20510 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -245,6 +245,11 @@ option(WITH_OPENVDB_BLOSC "Enable blosc compression for OpenVDB, only enable if
 option(WITH_OPENVDB_3_ABI_COMPATIBLE "Assume OpenVDB library has been compiled with version 3 ABI compatibility" OFF)
 mark_as_advanced(WITH_OPENVDB_3_ABI_COMPATIBLE)
 
+if(WITH_OPENVDB)
+    add_definitions(-DOPENVDB_3_ABI_COMPATIBLE)
+endif()
+
+
 # GHOST Windowing Library Options
 option(WITH_GHOST_DEBUG   "Enable debugging output for the GHOST library" OFF)
 mark_as_advanced(WITH_GHOST_DEBUG)
-- 
2.22.0

