From 4769595a02a7369e0aeef7336c7f3f86ae79dbe3 Mon Sep 17 00:00:00 2001
From: "Li, Guangli" <guangli.li@intel.com>
Date: Tue, 27 Aug 2019 17:09:09 +0800
Subject: [PATCH] add support for numpy

Signed-off-by: Li, Guangli <guangli.li@intel.com>
---
 build_files/cmake/macros.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build_files/cmake/macros.cmake b/build_files/cmake/macros.cmake
index c2f608d..71ff36e 100644
--- a/build_files/cmake/macros.cmake
+++ b/build_files/cmake/macros.cmake
@@ -1160,6 +1160,9 @@ function(find_python_package
       NAMES
         ${package}
       HINTS
+        "/usr/lib/"
+	"/usr/lib/python${PYTHON_VERSION}/"
+	"/usr/lib/python${_PY_VER_MAJOR}/"
         "${PYTHON_LIBPATH}/"
         "${PYTHON_LIBPATH}/python${PYTHON_VERSION}/"
         "${PYTHON_LIBPATH}/python${_PY_VER_MAJOR}/"
-- 
2.22.0

