--- blender-2.79b/build_files/cmake/Modules/FindPythonLibsUnix.cmake~	2018-03-23 15:22:25.000000000 +0000
+++ blender-2.79b/build_files/cmake/Modules/FindPythonLibsUnix.cmake	2018-09-18 15:03:44.493537429 +0000
@@ -38,7 +38,7 @@
   SET(PYTHON_ROOT_DIR $ENV{PYTHON_ROOT_DIR})
 ENDIF()
 
-SET(PYTHON_VERSION 3.5 CACHE STRING "Python Version (major and minor only)")
+SET(PYTHON_VERSION 3.7 CACHE STRING "Python Version (major and minor only)")
 MARK_AS_ADVANCED(PYTHON_VERSION)
 
 
